app/src/main/java/ch/sdp/vibester/activity/LyricsBelongGameActivity.kt

```diff
@@ -219,9 +208,9 @@ class LyricsBelongGameActivity : GameActivity() {
     /*
      * The following functions are helper for testing
      */
-    fun testCheckLyrics(ctx: Context, lyricToBeCheck: String, lyrics: String, gameManager: GameManager) {
+    /*fun testCheckLyrics(ctx: Context, lyricToBeCheck: String, lyrics: String, gameManager: GameManager) {
```

Comment:
should this be removed?

Reply:
It is not called because some tests are commented in the test, I will maybe work on that later so I prefer to keep it for now